example/android/app/build.gradle

```diff
@@ -34,7 +34,7 @@ android {
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
         applicationId "com.adobe.marketing.mobile.flutter.flutter_aepsdk_example"
-        minSdkVersion 19
+        minSdkVersion 21
```

Comment:
Is this the same minSDKVersion in our android native SDKs?

Reply:
Yes. Please check the Android 5.0 release notes: https://github.com/adobe/aepsdk-core-android/releases/tag/v3.0.0-core